javascript/ql/test/library-tests/InterProceduralFlow/TaintTracking.expected

```diff
@@ -18,6 +18,7 @@
 | global.js:5:22:5:35 | "also tainted" | global.js:9:13:9:22 | g(source1) |
 | global.js:5:22:5:35 | "also tainted" | global.js:10:13:10:22 | g(source2) |
 | nodeJsLib.js:1:15:1:23 | "tainted" | esClient.js:7:13:7:18 | nj.foo |
+| nodeJsLib.js:1:15:1:23 | "tainted" | esClient.js:10:13:10:17 | njFoo |
```

Comment:
Why do we get this flow?

Reply:
For reference here's `nodeJsLib.js`
```javascript
let source1 = "tainted";
let source2 = "tainted";

module.exports = source1;

exports.foo = source2;
```

The exported object itself is tainted, and we have a taint step through property reads, so there is a path step from `source1 -> (exports object).foo`. This PR models `import {foo} from 'nodeJsLib'` as a `PropRead` so that step is new.

It shouldn't matter in practice since module exports objects are never tainted.